agent/api/task/task_test.go

```diff
@@ -635,6 +635,183 @@ func TestPostUnmarshalTaskWithLocalVolumes(t *testing.T) {
 
 }
 
+// Slice of structs for Table Driven testing for sharing PID and IPC resources
+var namespaceTests = []struct {
```

Comment:
Please add tests for IPCMode None and Shareable as well.

Reply:
The customer will not be allowed to provide Shareable as an option. Shareable is internally set for the Pause container and is tested within a unit test.